from app.common.exceptions.repositories.base import BaseRepoError


class ConnectionRefusedRepoError(BaseRepoError):
    pass
